tests/data/placebo/test_datapipeline_reporting/datapipeline.DescribePipelines_1.json

```diff
@@ -0,0 +1,82 @@
+{
+    "status_code": 200, 
+    "data": {
+        "ResponseMetadata": {
+            "RetryAttempts": 0, 
+            "HTTPStatusCode": 200, 
+            "RequestId": "75644193-07e1-11e7-91e2-bfd5af170b83", 
+            "HTTPHeaders": {
+                "x-amzn-requestid": "75644193-07e1-11e7-91e2-bfd5af170b83", 
+                "date": "Mon, 13 Mar 2017 11:37:37 GMT", 
+                "content-length": "849", 
+                "content-type": "application/x-amz-json-1.1"
+            }
+        }, 
+        "pipelineDescriptionList": [
+            {
+                "fields": [
```

Comment:
ick, thats an odd api design on aws part, so we might need to just implement an augment method here instead of batch details spec, so that we can take this structure and map it back to something a bit more sane for ease of filtering.